es-core/src/components/NinePatchComponent.h

```diff
@@ -37,9 +37,10 @@ class NinePatchComponent : public GuiComponent
 
 	virtual void applyTheme(const std::shared_ptr<ThemeData>& theme, const std::string& view, const std::string& element, unsigned int properties) override;
 
-private:
-	Vector2f getCornerSize() const;
+	const Vector2f& getCornerSize() const;
+	void setCornerSize(const Vector2f& size);
```

Comment:
Small question here, can't we do : 
```c++
void setCornerSize(Vector2f size);
inline void setCornerSize(float sizeX, float sizeY) { setOrigin(Vector2f(sizeX, sizeY);
```

Because the size is stored as a Vector2f, if we do your solution, that mean we will take both x and y component of a Vector2f to build another Vector2f out of it. So, why were all the others designed that way ?

Reply:
I was just asking this out of curiosity. Sure, let's do it like @jrassa said and move ahead on this PR.